src/test/lambda/explorer/lambdaFunctionNode.test.ts

```diff
@@ -4,26 +4,26 @@
  */
 
 import * as assert from 'assert'
-import { Lambda } from 'aws-sdk'
+import { CloudWatchLogs } from 'aws-sdk'
 import * as os from 'os'
-import { LambdaFunctionNode } from '../../../lambda/explorer/lambdaFunctionNode'
+import { LogGroupNode } from '../../../CloudWatchLogs/explorer/logGroupNode'
 import { ext } from '../../../shared/extensionGlobals'
 import { TestAWSTreeNode } from '../../shared/treeview/nodes/testAWSTreeNode'
 import { clearTestIconPaths, IconPath, setupTestIconPaths } from '../../shared/utilities/iconPathUtils'
 
-describe('LambdaFunctionNode', () => {
+describe('LogGroupNode', () => {
```

Comment:
Why are these edits in the lambda function node test file?

Reply:
Because I copy/pasted and edited the source instead of the copy -_-